lib/cosmos/system/target.rb

```diff
@@ -90,6 +96,8 @@ def initialize(target_name, substitute_name = nil, path = nil, target_filename =
       @routers = []
       @cmd_cnt = 0
       @tlm_cnt = 0
+      @cmd_unique_id_mode = false
+      @tlm_unique_id_mode = false
```

Comment:
So the default is that cmd/tlm do NOT have unique ID locations and thus we do the new fast lookup. If this is not the case you add CMD_UNIQUE_ID_MODE or TLM_UNIQUE_ID_MODE to your target.txt file. Correct? Once we release this I'll update the website and note this.

Reply:
That is correct.  The speed improvement is very beneficial and is the common case.  Is backwards breaking however so the next release will be COSMOS 4.4